specification/subscription/resource-manager/Microsoft.Subscription/preview/2019-10-01-preview/subscriptions.json

```diff
@@ -117,6 +130,56 @@
           }
         }
       }
+    },
+    "/subscriptions/{subscriptionId}/providers/Microsoft.Subscription/PurchaseSupport": {
```

Comment:
Technically this can be used to purchase, cancel and convert - can we call it something more generic - `configureSupport` for example?